main.js

```diff
@@ -1,193 +1,177 @@
-import odredi from '@pionir/detektor.js';
+import odredi from '@pionir/detektor-js';
```

Comment:
`import odredi from '@pionir/detektor.js';` 

Doesn't exist on npm registry while '@pionir/detektor-js' does and is valid one.